assets/json/menu.json

```diff
@@ -0,0 +1,641 @@
+          "data": {
+            "font": "felt32",
+            "text": "How to Play",
+            "foreground": [255, 255, 255, 255],
+            "background": [0, 0, 0, 0],
+            "size": [300, 80],
+            "anchor": [0.5, 0.5],
+            "visible": false
+          },
+          "layout": {
+            "x_anchor": "center",
+            "y_anchor": "middle",
+            "x_offset": 0.35,
+            "y_offset": 0.3
+          }
+        },
+
+        "joingameback": {
+          "comment": "The back button to return to main menu.",
+          "type": "Widget",
+          "data": {
+            "key": "backbutton"
+          },
+          "layout": {
+            "x_anchor": "center",
+            "y_anchor": "middle",
+            "x_offset": 0.05,
+            "y_offset": 0.38
+          }
+        },
+        "joingamejoinbutton": {
+          "comment": "The 'Join Game' button to join the game lobby.",
+          "type": "Widget",
+          "data": {
+            "key": "menubutton",
+            "variables": {
+              "size": [300, 75],
+              "text": "Join Game",
+              "size2": [300, 75],
+              "text2": "Join Game"
+            }
+          },
+          "layout": {
+            "x_anchor": "center",
+            "y_anchor": "middle",
+            "x_offset": 0.25,
+            "y_offset": -0.15
+          }
+        },
+        "joingameroomidinput": {
+          "comment": "The Game Room Id text input in Join Game page",
+          "type": "TextField",
+          "data": {
+            "font": "felt32",
+            "text": "",
+            "foreground": [255, 255, 255, 255],
+            "background": [0, 0, 0, 255],
+            "size": [300, 80],
+            "anchor": [0.5, 0.5],
+            "visible": false
+          },
+          "layout": {
+            "x_anchor": "center",
+            "y_anchor": "middle",
+            "x_offset": 0.25,
+            "y_offset": 0.05
+          }
+        },
+
+        "gamelobbyback": {
+          "comment": "The back button to return to main menu.",
+          "type": "Widget",
+          "data": {
+            "key": "backbutton"
+          },
+          "layout": {
+            "x_anchor": "center",
+            "y_anchor": "middle",
+            "x_offset": 0.05,
+            "y_offset": 0.38
+          }
+        },
+        "gamelobbyroomidlabel": {
+          "comment": "The text label showing the current game room id",
+          "type": "Label",
+          "data": {
+            "font": "felt32",
+            "text": "",
+            "foreground": [255, 255, 255, 255],
+            "background": [0, 0, 0, 0],
+            "size": [300, 80],
+            "anchor": [0.5, 0.5],
+            "visible": false
+          },
+          "layout": {
+            "x_anchor": "center",
+            "y_anchor": "middle",
+            "x_offset": 0.35,
+            "y_offset": 0.4
+          }
+        },
+        "gamelobbyplayerlabel1": {
```

Comment:
Later on we will have to move populating the list of players in the lobby programmatically with their names.